pom.xml

```diff
@@ -463,6 +463,7 @@
                   <arg value="-DCUDF_USE_PER_THREAD_DEFAULT_STREAM=${CUDF_USE_PER_THREAD_DEFAULT_STREAM}" />
                   <arg value="-DCUDF_LARGE_STRINGS_DISABLED=ON"/>
                   <arg value="-DKvikIO_REMOTE_SUPPORT=OFF"/>
+                  <arg value="-DCUDF_KVIKIO_REMOTE_IO=OFF"/>
```

Comment:
Maybe remove `KvikIO_REMOTE_SUPPORT`? Since `CUDF_KVIKIO_REMOTE_IO` disables it.

Reply:
this is for preemptive change, current cudf ref in JNI repo is still require this flag until https://github.com/rapidsai/cudf/pull/17291 is merged